tests/unit/algos/set/ArrayUtils.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const { indexOf, indexAtOrBelow, symDiff } = require('../../../../lib/algos/set/ArrayUtils');
+const crypto = require('crypto');
+
+describe('ArrayUtils', () => {
+    it('indexOf', () => {
+        const numOps = 10000;
```

Comment:
This test is definitely useful, I'd also add a 0, 1 and 2-item array test for both `indexOf` and `indexAtOrBelow`